src/main/java/net/goldiriath/plugin/game/mobspawn/MobSpawnProfile.java

```diff
@@ -37,20 +38,23 @@
     private final Logger logger;
     //
     @Getter
-    private final Set<PotionEffect> effects;
+    private String customName;
+    @Getter
+    private long timeThreshold;
     @Getter
     private EntityType type;
     @Getter
     private int level;
     @Getter
-    private String customName;
+    private int damage;
     @Getter
-    private long timeThreshold;
+    private int health;
     @Getter
-    private ItemStack item;
-    private long spawnThreshold;
+    private MobTier lootTier;
     @Getter
-    private ItemStack carryItem;
+    private LootProfile lootProfile;
+    @Getter
+    private ItemStack hand;
```

Comment:
@cashenkes Currently this is not supported. Do we need this functionality for Alpha?